atlasdb-impl-shared/src/main/java/com/palantir/atlasdb/transaction/impl/SnapshotTransaction.java

```diff
@@ -1579,6 +1579,8 @@ private void waitForCommitToComplete(Iterable<Long> startTimestamps) {
                     watch.elapsed(TimeUnit.MILLISECONDS), tableRef);
         }
 
+        log.trace("Getting commit timestamps for {} start timestamps in response to read from table {}",
+                gets.size(), tableRef.toString());
```

Comment:
Nit: don't really need the toString on tableRef as that will cause us to always generate the string even if we're not tracing. You can just pass `tableRef` and it will only be stringified when it is logged.